sw/device/tests/power_virus_systemtest.c

```diff
@@ -422,16 +368,21 @@ static void configure_pinmux(void) {
   CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIoa5,
                                         kTopEarlgreyPinmuxOutselUart1Tx));
 
-  // UART2:
-  //    RX on IOB4
-  //    TX on IOB5
-  CHECK_DIF_OK(dif_pinmux_input_select(&pinmux,
-                                       kTopEarlgreyPinmuxPeripheralInUart2Rx,
-                                       kTopEarlgreyPinmuxInselIob4));
-  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob4,
-                                        kTopEarlgreyPinmuxOutselConstantHighZ));
-  CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIob5,
-                                        kTopEarlgreyPinmuxOutselUart2Tx));
+  // Apply this configuration only for the FPGA.
+  // For the simulation, apply the config in configure_pinmux_sim().
+  if (kDeviceType == kDeviceFpgaCw305 || kDeviceType == kDeviceFpgaCw310) {
```

Comment:
spi_device_agent_1 is connected to a fixed set of MIOs inside the chip_if.sv. Therefore, the settings for the simulation should be compatible with chip_if.sv.

In configure_pinmux_sim() below, I update the sim config for UART2, SPI HOST1, and  PWM. Therefore, the corresponding settings here (in pinmux_configure()) should be exclusive to FPGA.